src/components/Output/index.tsx

```diff
@@ -87,6 +168,7 @@ export default class Output extends Component<Props, State> {
     if (event.type !== 'wheel' && targetEl.closest('.' + twoUpHandle)) return;
     // If we've already retargeted this event, let it through.
     if (this.retargetedEvents.has(event)) return;
+    if (event.type === 'pointerdown') this.pinchZoomLeft.focus();
```

Comment:
It removes focus from the buttons. The pinch-zoom component prevents the default pointerdown action, which means focussed buttons aren't blurred when interacting with pinch-zoom.  I'm going to remove this since I think it might be what causes the weird zooming issue I noticed last week.